frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/Indices/Indices.tsx

```diff
@@ -1,30 +1,56 @@
 import type { Indices as IndicesType } from '@liam-hq/db-structure'
-import { Hash } from '@liam-hq/ui'
-import type { FC } from 'react'
+import { ChevronDown, ChevronUp, Hash, IconButton } from '@liam-hq/ui'
+import clsx from 'clsx'
+import { type FC, type MouseEvent, useState } from 'react'
 import styles from './Indices.module.css'
 
 type Props = {
   indices: IndicesType
 }
 
 export const Indices: FC<Props> = ({ indices }) => {
+  const [isClosed, setIsClosed] = useState(false)
+  const handleClose = (event: MouseEvent) => {
+    event.stopPropagation()
+    setIsClosed(!isClosed)
+  }
+  const handleKeyDown = (event: { key: string }) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      setIsClosed(!isClosed)
+    }
+  }
+
   return (
-    <div className={styles.wrapper}>
-      <div className={styles.header}>
+    <>
+      <div
+        className={styles.header}
+        // biome-ignore lint/a11y/useSemanticElements: Implemented with div button to be button in button
+        role="button"
+        tabIndex={0}
+        onClick={handleClose}
+        onKeyDown={handleKeyDown}
+      >
         <div className={styles.iconTitleContainer}>
           <Hash width={12} />
           <h2 className={styles.heading}>Indexes</h2>
         </div>
+        <IconButton
+          icon={isClosed ? <ChevronDown /> : <ChevronUp />}
+          tooltipContent={isClosed ? 'Open' : 'Close'}
+          onClick={handleClose}
+        />
       </div>
       {Object.keys(indices).length !== 0 && (
-        <ul className={styles.list}>
-          {Object.entries(indices).map(([key, index]) => (
-            <li key={key} className={styles.listItem}>
-              {index.name}
-            </li>
-          ))}
-        </ul>
+        <div className={clsx(!isClosed && styles.visible, styles.listWrapper)}>
```

Comment:
I had to add an extra ``div`` to fix an animation issue caused by the ``padding``.
If we add a class ``visible`` to the ``list`` class, it would look like this:


https://github.com/user-attachments/assets/b7365149-98ea-4578-a6ed-bf3d629b06ff